share/fstab

```diff
@@ -23,7 +13,9 @@
 #       documentation for details on setting a label. To obtain the UUID, use
 #       the blkid(8) command.
 
-#LABEL=boot		/boot		ext4		noauto,noatime	1 2
-#UUID=58e72203-57d1-4497-81ad-97655bd56494		/		ext4		noatime		0 1
+# <fs>			<mountpoint>	<type>		<opts>		<dump> <pass>
+
+#LABEL=boot		/boot		ext4		defaults	1 2
```

Comment:
Should `/boot/` remain using `ext4`? (I have no idea). I know e.g. GRUB has XFS support.

Reply:
I'll keep it as ext4 for now because there's a constraint of at least 300MB for XFS filesystems, although I did recently change the handbook to say 1GB is a sensible default there now for /boot.